src/components/ChatEntry.js

```diff
@@ -1,22 +1,49 @@
 import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
 
 const ChatEntry = (props) => {
+  const onLikeButtonClick = () => {
+    const updatedEntry = {
+      id: props.id,
+      sender: props.sender,
+      body: props.body,
+      timeStamp: props.timeStamp,
+      liked: !props.liked,
+    };
+
+    props.onUpdate(updatedEntry);
+  };
+
+  const likedStatus = !props.liked ? '🤍' : '❤️';
+
   return (
-    <div className="chat-entry local">
-      <h2 className="entry-name">Replace with name of sender</h2>
+    <div
+      className={`chat-entry ${
+        props.sender === 'Vladimir' ? 'local' : 'remote'
+      }`}
```

Comment:
Great way to update just the local/remote part of this class info. But what if the chat participants were other senders? Could we determine the "local sender" higher up and then pass that information down so that we can avoid hard-coding a name?